src/main/java/com/online/cinema/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -31,7 +31,9 @@ public MovieSession get(Long id) {
 
     @Override
     public List<MovieSession> findAvailableSessions(Long movieId, LocalDate date) {
+
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            logger.info("Trying to get available sessions");
```

Comment:
it is better to have logger object in each of the classes
for logs to be more informative